### PR TITLE
Weather added

### PR DESCRIPTION
**Pull request by prudyvusandriana**
Updated info in DB
Created weather in arrival cities models and handlers

### CODE REVIEW

File: src/service/router.go [hunk 1]
```diff
@@ -120,6 +121,12 @@ func NewRouter() *mux.Router {
 	apiV1.Handle("/trips/{id}", common.MethodHandler(map[string]http.Handler{
 		http.MethodGet: http.HandlerFunc(trips.GetTripHandler),
 	}))
+	apiV1.Handle("/train/weather/{id}", common.MethodHandler(map[string]http.Handler{
```
Comment (YaromyrO) [inline src/service/router.go]: better trains and flights

**Comment by Nastya-Kruglikova** [comment]
Are all files added?

File: src/model/weather.go [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	if err != nil {
+		return err
+	}
+	defer r.Body.Close()
+
+	return json.NewDecoder(r.Body).Decode(target)
+}
+
+type Forecast struct {
+	Id      int
```
Comment (YaromyrO) [inline src/model/weather.go]: better ID

File: src/service/weather/weather.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+import (
+	"github.com/Nastya-Kruglikova/cool_tasks/src/model"
+	"github.com/Nastya-Kruglikova/cool_tasks/src/service/common"
+	"github.com/gorilla/mux"
+	"github.com/satori/go.uuid"
+	"net/http"
+)
+
+func GetWeatherByTrainIdHandler(w http.ResponseWriter, r *http.Request) {
```
Comment (YaromyrO) [inline src/service/weather/weather.go]: also better GetWeatherByTrainIDHandler

**Review by YaromyrO — APPROVED** [review]

**Review by Pliama — APPROVED** [review]

**Pull request merged**